docs/dev/python/table-api-users-guide/table_environment.md

```diff
@@ -0,0 +1,833 @@
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Execute/Explain Jobs
+
+These APIs are used to explain/execute jobs. Note that the API `execute_sql` can also be used to execute job.
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 20%">APIs</th>
+      <th class="text-center">Description</th>
+      <th class="text-center" style="width: 10%">Docs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>explain_sql(stmt, *extra_details)</strong>
+      </td>
+      <td>
+        Returns the AST of the specified statement and the execution plan.
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.explain_sql">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>create_statement_set()</strong>
+      </td>
+      <td>
+        Create a StatementSet instance which accepts DML statements or Tables.
+        It can be used to execute a multi-sink job. 
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.create_statement_set">link</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<big><strong>Deprecated APIs</strong></big>
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 20%">APIs</th>
+      <th class="text-center">Description</th>
+      <th class="text-center" style="width: 10%">Docs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>explain(table=None, extended=False)</strong>
+      </td>
+      <td>
+        Returns the AST of the specified Table API and SQL queries and the execution plan to compute
+        the result of the given `Table` object or multi-sinks plan.
+        If you use the "insert_into" or "sql_update" method to emit data to multiple sinks, you can use this
+        method to get the plan.
+        It can be replaced by "explain_sql", `Table`.explain or `StatementSet`.explain().
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.explain">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>execute(job_name)</strong>
+      </td>
+      <td>
+        Triggers the program execution. The environment will execute all parts of the program.
+        If you use the "insert_into" or "sql_update" method to emit data to sinks, you can use this
+        method trigger the program execution.
+        This method will block the client program until the job is finished/canceled/failed.
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.execute">link</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Create/Drop User Defined Functions
+
```

Comment:
Users may be confused why there are so many kinds of functions. Could you add a link to https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/functions/ which explains more details about the type of functions?